### PR TITLE
Add new Cornerstone Learning vILT connector

### PR DESCRIPTION
**Pull request by mpequenocamarero**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ X] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [ X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ X] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [X ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by mpequenocamarero** [comment]
@microsoft-github-policy-service agree company="Cornerstone OnDemand"

File: certified-connectors/Cornerstone Learning vILT/Readme.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+## Cornerstone Learning vILT Connector
+The custom connector streamlines vILT integration by leveraging customers' Azure AD tenant environments, ensuring data integrity and simplifying authentication. No external Graph API permissions are needed, enhancing security. This versatile connector facilitates low-code solutions for diverse scenarios. By incorporating this custom connector, the full potential of Microsoft Teams API in the case of MS Teams integration is unlocked seamlessly within customers' infrastructure.
```
Comment (vmanoharas) [inline certified-connectors/Cornerstone Learning vILT/Readme.md:2]: please format the readme.md file according to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md). All of the required headings should be present.

File: certified-connectors/Cornerstone Learning vILT/apiProperties.json [hunk 2]
```diff
@@ -0,0 +1,167 @@
+{
+	"properties": {
```
Comment (vmanoharas) [inline certified-connectors/Cornerstone Learning vILT/apiProperties.json]: Kindly correct the apiProperties.json file as per the schema here https://github.com/microsoft/PowerPlatformConnectors/blob/dev/schemas/paconn-apiProperties.schema.json.

Please refer this validation error message - ##[error]@{level=Error; message=JSON is valid against no schemas from 'oneOf'.; valuePath=line: 5 position: 11 path: '[0].properties.connectionParameters.url'; value=}

**Review by vmanoharas — COMMENTED** [review]
Hello @mpequenocamar, 

Please resolve review comments.

File: certified-connectors/Cornerstone Learning vILT/Readme.md (hunk 1 above)
Comment (mpequenocamarero) [reply certified-connectors/Cornerstone Learning vILT/Readme.md:2]: Corrected in last commit. Thanks!

File: certified-connectors/Cornerstone Learning vILT/apiProperties.json (hunk 2 above)
Comment (mpequenocamarero) [reply certified-connectors/Cornerstone Learning vILT/apiProperties.json]: Corrected in last commit. Thanks!

File: certified-connectors/Cornerstone Learning vILT/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,1143 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"title": "Cornerstone Learning vILT",
+		"version": "1.0",
+		"description": "The custom connector streamlines vILT integration by leveraging customers' Azure AD tenant environments, ensuring data integrity and simplifying authentication. No external Graph API permissions are needed, enhancing security. This versatile connector facilitates low-code solutions for diverse scenarios. By incorporating this custom connector, the full potential of Microsoft Teams API in the case of MS Teams integration is unlocked seamlessly within customers' infrastructure.",
+		"contact": {
```
Comment (vmanoharas) [inline certified-connectors/Cornerstone Learning vILT/apiDefinition.swagger.json:7]: Please add the url to contact, refer this template - https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiDefinition.swagger.json

**Review by vmanoharas — COMMENTED** [review]
Hello @mpequenocamar

Can you please resolve review comment?

File: certified-connectors/Cornerstone Learning vILT/apiDefinition.swagger.json (hunk 3 above)
Comment (mpequenocamarero) [reply certified-connectors/Cornerstone Learning vILT/apiDefinition.swagger.json:7]: I understand that you are referring to the email tag? If so, I have added as value NA, we don't have a support email, users have to log a ticket in our platform.

**Comment by mpequenocamarero** [comment]
Disregards my last comment. I understood the issue was in a blank in the URL. Reviewed the full swagger.

File: certified-connectors/Cornerstone Learning vILT/apiDefinition.swagger.json (hunk 3 above)
Comment (vmanoharas) [reply certified-connectors/Cornerstone Learning vILT/apiDefinition.swagger.json:7]: Kindly add email tag as well, this is needed for future communication to add into our internal tool.

**Review by vmanoharas — COMMENTED** [review]
Hello @mpequenocamar

Kindly review below swagger errors and fix them. Additionally, please fix the review comments. 


##[error]The path template has been changed for the operation. ValuePath: operation/Launch-session-subscribe Source: NewSwagger
##[error]The path template has been changed for the operation. ValuePath: operation/createSessionResponse Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/modifySessionResponse Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/modifyInstructorResponse Source: NewSwagger

**Comment by mpequenocamarero** [comment]
Regarding the swagger we have modified the endpoints of the API (eg from modifyInstructorResponse to updateInstructorResponse) and redeployed the custom connector in power apps. Please advise if there is any other thing to correct or if maybe a new PR is needed as this are significant changes.

**Comment by vmanoharas** [comment]
> Regarding the swagger we have modified the endpoints of the API (eg from modifyInstructorResponse to updateInstructorResponse) and redeployed the custom connector in power apps. Please advise if there is any other thing to correct or if maybe a new PR is needed as this are significant changes.

Thank you @mpequenocamarero, all validations are passed, I will approve this connector.

**Review by vmanoharas — APPROVED** [review]
Hello @mpequenocamarero 
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to Connector certification portal](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#update-submission-prior-to-certification) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in [Connector certification portal](https://connectorcertification.azurewebsites.net/) . Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in [Connector certification portal](https://connectorcertification.azurewebsites.net/), please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Pull request merged**